tests/pytorch/test_models_classification_pt.py

```diff
@@ -89,3 +94,45 @@ def test_crop_orientation_model(mock_text_box):
     text_box_270 = np.rot90(text_box_0, 3)
     classifier = classification.crop_orientation_predictor("mobilenet_v3_small_orientation", pretrained=True)
     assert classifier([text_box_0, text_box_90, text_box_180, text_box_270]) == [0, 1, 2, 3]
+
+
+@pytest.mark.parametrize(
+    "arch_name, input_shape, output_size",
+    [
+        ["vgg16_bn_r", (3, 32, 32), (126,)],
+        ["resnet18", (3, 32, 32), (126,)],
+        ["resnet31", (3, 32, 32), (126,)],
+        ["resnet34", (3, 32, 32), (126,)],
+        ["resnet34_wide", (3, 32, 32), (126,)],
+        ["resnet50", (3, 32, 32), (126,)],
+        ["magc_resnet31", (3, 32, 32), (126,)],
+        ["mobilenet_v3_small", (3, 32, 32), (126,)],
+        ["mobilenet_v3_large", (3, 32, 32), (126,)],
+        ["mobilenet_v3_small_orientation", (3, 128, 128), (4,)],
+    ],
+)
+def test_models_onnx_export(arch_name, input_shape, output_size):
+    # Model
+    batch_size = 2
+    model = classification.__dict__[arch_name](pretrained=True).eval()
+    dummy_input = torch.rand((batch_size, *input_shape), dtype=torch.float32)
+    try:
+        with tempfile.TemporaryDirectory() as tmpdir:
+            # Export
+            torch.onnx.export(
+                model,
+                dummy_input,
+                os.path.join(tmpdir, "model.onnx"),
+                input_names=['input'],
+                output_names=['logits'],
+                dynamic_axes={'input': {0: 'batch_size'}, 'logits': {0: 'batch_size'}},
+                export_params=True, opset_version=13, verbose=False
+            )
+            # Inference
+            quantize_qat(os.path.join(tmpdir, "model.onnx"), os.path.join(tmpdir, "model.quant.onnx"))
+            ort_session = onnxruntime.InferenceSession(os.path.join(tmpdir, "model.quant.onnx"))
+            ort_outs = ort_session.run(['logits'], {'input': dummy_input.numpy()})
+            assert isinstance(ort_outs, list) and len(ort_outs) == 1
+            assert ort_outs[0].shape == (batch_size, *output_size)
+    except RuntimeError:
+        assert False
```

Comment:
Why do you put a try except in this test ?

Reply:
Good catch  it's removed :+1: